src/flamegraph/mod.rs

```diff
@@ -881,6 +902,13 @@ fn filled_rectangle<W: Write>(
             "fg:x" => &buffer[x_samples],
             "fg:w" => &buffer[width_samples]
         ));
+
+        if let Some(stroke_color) = stroke_color {
+            bytes_start.extend_attributes(args!(
+                "stroke" => stroke_color,
+                "stroke_width" => "1"
+            ));
+        }
```

Comment:
It feels unfortunate to add this attribute to every single frame — can we instead just add it to the style sheet instead only when the option is set?

Reply:
Yes we can. I added them to `#frames > g > rect` like `@nagisa` was doing in their blog.